docs/docs-content/clusters/data-center/nutanix/register-nutanix-cloud.md

```diff
@@ -0,0 +1,263 @@
+
+1. From your terminal, issue a command such as `ls -l` to list the files and confirm you have the following YAML templates:
+
+    - infrastructure-components.yaml
+    - cloudClusterTemplate.yaml    
+    - controlPlanePoolTemplate.yaml
+    - workerPoolTemplate.yaml
+
+
+2. Ensure each template contains objects as listed in the table.
+
+  | **Templates**                  | **Objects**           |
+  |--------------------------------|------------------------|
+  | **cloudClusterTemplate.yaml**    | ConfigMap<br />Secret<br />Cluster<br />NutanixCluster<br />MachineHealthCheck | 
+  | **controlPlanePoolTemplate.yaml**| KubeadmControlPlane<br />NutanixMachineTemplate |
+  | **workerPoolTemplate.yaml**     | KubeadmConfigTemplate<br />MachineDeployment<br />NutanixMachineTemplate |
+
+3. Open each file and verify that all occurrences of `${NAMESPACE}` are removed. 
+
+4. In the **controlPlanePoolTemplate.yaml** file, ensure `${CLUSTER_NAME}-cp-0` for the KubeadmControlPlane and NutanixMachineTemplate objects have the same name.
+
+5. Verify parameters are modified as described for each template in steps 6 and 7 of [Customize YAML Configuration Files](#customize-yaml-configuration-files).
+
+## Register the Cloud
+
+Follow the steps below from your terminal to set the environment variables and invoke the APIs required to register a Nutanix cloud to Palette. Alternatively, you can use an API platform such as [Postman](https://www.postman.com/).
+
+:::caution
+
+The logo file must not exceed 100KB in size. To ensure image quality ensure at least one dimension in either width or height is 40 pixels. It is preferable that the image be transparent. 
+
+:::
+
+1. Export the URL of your self-hosted Palette or VerteX instance and the cloud type as environment variables. Additionally, export the path to the YAML templates and logo file.
+
+  ```bash
+  export ENDPOINT="https://palette.example.com"
+  export CLOUD_TYPE="nutanix"
+  export cloudLogo="/path/to/the/file/cloud-logo.png"
+  export infraComponents="/path/to/the/file/infrastructure-components.yaml"
+  export cloudClusterTemplate="/path/to/the/file/cloudClusterTemplate.yaml"
+  export  controlPlanePoolTemplate="/path/to/the/file/controlPlanePoolTemplate.yaml"
+  export workerPoolTemplate="/path/to/the/file/workerPoolTemplate.yaml"
+  ```
+
+  :::caution
+  The CLOUD_TYPE variable value must be set as `nutanix`, as this value will be used for the `name` value in the `/v1/clouds/cloudTypes/register` cloud registration API in step 4 below. Setting `name` as `nutanix` will make the out-of-the-box **Nutanix CSI** pack available to users when they create a cluster profile in Palette. 
+  :::
+
+2. To acquire system administrator credentials, use the `/v1/auth/syslogin` endpoint. Issue the `curl` command below and ensure you replace the credentials with your system console credentials.
+
+  ```bash
+  curl --insecure --location "$ENDPOINT/v1/auth/syslogin" \
```

Comment:
We should **not** provide examples where we have `-k` or `--insecure` in our docs, righ @karl-cardenas-coding ?

Reply:
Removed `--insecure`.

Also changed double quotes to single quotes in the --data block, as discussed.